src/main/java/edu/kis/vh/nursery/list/Node.java

```diff
@@ -1,12 +1,21 @@
 package edu.kis.vh.nursery.list;
 
 public class Node {
+    /**
+     * protected Node prev - poprzedni Node listy
+     * protected Node next - następny Node listy
+     * private final int value - wartość, którą przechowuje dany Node
+     * public Node(int i) - kontruktor Noda, który odrazu go inicjalizuje
+     * public int getValue() - zwraca wartość, którą przechoowuje Node
+     */
+    final private int value;
+    protected Node prev, next;
```

Comment:
Task 2 in "accessory i hermetyzacja" concerned only 'total' getter.
Lack of encapsulation in Node class (Task 3 in "accessory i hermetyzacja")